roles/kubernetes/node/defaults/main.yml

```diff
@@ -136,7 +136,7 @@ kubelet_custom_flags: []
 
 # If non-empty, will use this string as identification instead of the actual hostname
 kube_override_hostname: >-
-  {%- if cloud_provider is defined and cloud_provider in ['aws'] -%}
+  {%- if cloud_provider is defined -%}
```

Comment:
Same.
(Incidentally, we should not have two definitions for `kube_override_hostname`, this should be moved in a more general place... But that's out of scope of this PR)

Reply:
Exactly. XD